ldm/dream/conditioning.py

```diff
@@ -0,0 +1,93 @@
+'''
+This module handles the generation of the conditioning tensors, including management of
+weighted subprompts.
+'''
+import re
+import torch
+
+class Conditioning():
+    def __init__(self, model, log_tokens=False):
```

Comment:
This class is only ever instantiated to call `get_uc_and_c`, once, and then discarded. Seems like it would be nicer to just export `get_uc_and_c` directly, and have it take the `model` and `log_tokens` parameters itself, instead of wrapping it up in a class.

Reply:
Ok, fixed it up in the recommended way.